test/unit/lib/email.js

```diff
@@ -0,0 +1,18 @@
+import { assert } from 'chai';
+import StudentStore from '../../../.dist/store/student-store';
+import * as email from '../../../.dist/lib/email';
+import sinon from 'sinon';
+
+describe("Email system", () => {
+    it("Sends an email to the correct students", () => {
+        let getStudentsWithDueItems = sinon.stub(StudentStore, "getStudentsWithDueItems");
+
+        getStudentsWithDueItems.returns([
+            {email: 'trevinhofmann@gmail.com'}
```

Comment:
Changing the amount of emails that this method returns makes the 'should get students with soon to be due items' test fail. Might have something to do with you not restoring the spy?